distributed/utils_test.py

```diff
@@ -104,6 +104,22 @@ def is_debugging() -> bool:
 _offload_executor.submit(lambda: None).result()  # create thread during import
 
 
+class _Watch:
+    def __init__(self, duration: float):
+        self.duration = duration
+        self.started_at = None
+
+    def start(self) -> None:
```

Comment:
I think it would be better to start immediately rather than have a two phase construct and start